course_inventory/inventory.py

```diff
@@ -268,6 +336,11 @@ def run_course_inventory() -> Sequence[Dict[str, Union[ValidDataSourceName, pd.T
 
     if CREATE_CSVS:
         # Generate CSV Output
+        if len(new_term_df) > 0:
```

Comment:
The term.csv file only contains new term information -- is this as desired?

Reply:
You tell me; I can change it if you want. It made sense to me to use the `append_table` logic here; we're not using an append table with CSV generation elsewhere yet. I could also just not include this in the CSV generation.

Reply:
I’ll rethink this in the morning. It may make more sense to not treat this as an append table.

Reply:
@zqian, after doing some thinking, I decided it was simpler and less confusing to drop this table as well. This should also make the `term_df` output more regular. See [this commit](https://github.com/tl-its-umich-edu/course-inventory/pull/115/commits/32500297adc106142db3cffbdee96a2edbaa7e14).

I'm also thinking that this will simplify what I'm currrently thinking of as the solution for #118 (a `DELETE` with `JOIN`s and a `WHERE`, see issue for SQL).

Reply:
Gonna go with this approach for now, we can always revisit.